### ISSUE TITLE
Sound not disabled when starting without focus

### ISSUE THREAD

**Issue by jensj12** [OpenRCT2/OpenRCT2#2287]
Enable 'disable sound on focus loss', start the game and go to another window before the game is started up. Even though the game doesn't have focus, sound is enabled. If you lose focus again after going back to the game the sound is disabled like it should.

**Comment by zsilencer** [comment]
looks like normal behavior for windows, even starcraft 2 does this.

**Comment by PFCKrutonium** [comment]
Actually I think it's just a bug that usually doesn't get addressed - normally you look for a Window Minimised Event, in this case you miss the event, and should probably check the Window status after init finishes.

**Comment by HaasJona** [comment]
I agree with @zsilencer and don't think this is a bug. First, the setting is labelled 'disable sound on focus loss' so if there is no focus to begin with and therefore no focus loss, then I don't see why the sound should be disabled. Also if you have a slow PC and you start the game in the background then it's helpful to know when the game is loaded fully because that is when you're getting sound. If you start the game you usually want to have sound (and if not you usually want to disable sound completely anyway) so I think the current behaviour makes sense.

**Comment by mrtnptrs** [comment]
I agree with @zsilencer and @HaasJona and this could also be treated as a very minor "bug". My advice would be to close this issue. Could anyone do this or say why that wouldn't be good? :)

**Comment by tupaschoal** [comment]
Still happens on v0.3.1

**Comment by Lukasz-Pe** [comment]
Could you please lend me a hand?
I am trying to pinpoint the place where main window steals focus (and is brought to front). I intend to change behaviour so that the window will stay minimised and without focus and in the same place I want to set the volume to 0 and thus getting expected behaviour without the need for additional focus changes after window got focus and is the top most one.
I am testing with OpenGL and on fullscreen.

**Comment by 733737** [comment]
I cannot replicate this

**Comment by jensj12** [comment]
I can still reproduce the issue on 6d97bbd, though it sometimes mutes after about 1 second (depending on whether the other window is fullscreen, as far as I've tested).

**Comment by MattBDev** [comment]
I think the issue is fixed even if there is a short delay before the music pauses.
